src/qt/transactiontablemodel.cpp

```diff
@@ -734,7 +734,7 @@ void TransactionTablePriv::ShowProgress(const std::string &title, int nProgress)
         }
         for (unsigned int i = 0; i < vQueueNotifications.size(); ++i)
         {
-            if (vQueueNotifications.size() - i <= 10) {
+            if (vQueueNotifications.size() == 10 + 1 + i) {
```

Comment:
In commit "qt: Enqueue setProcessingQueuedTransactions(false) only once" (5ff42b72e93508dfae41e082bbbc7c7ee4aacf6b)

This is kind of hard to follow, and not 100% sure I understand it. IIUC, calling `setProcessingQueuedTransactions` with `false` argument turns on balloons, calling it with `true` argument turns off balloons.

Previously if queue size more than `10`, balloons were turned off before looping through transactions. Then for every transaction starting at `i >= size-10` (i.e. for the most recent 10 transactions) balloons were turned on before updating each transactions.

Now balloons are turned on at `i == size-11` so an extra balloon will be shown. (For example if size is `11`, balloons are now turned on at `i=0` instead of `i=1`).

If this is right, I'd make a few suggestions:

1. Update commit description to say what motivation for the change is: e.g. that this is a optimization not supposed to change behavior. Not a bugfix or something else.
2. Drop `+ 1` term which does not seem correct
3. Add comment before if statement like `// Re-enable balloons for the most recent 10 transactions`
4. Maybe write expression as `i == size-10` instead of `size == 10 + i`, which seems more obscure

Reply:
Ok, I'll just drop this commit, it's unrelated anyway to the PR goal. This ballon filtering should be done elsewhere anyway.

Reply:
Removed 5ff42b7,